canonical-web-and-design/ubuntu.com
# Fix footer middot alignment

## Done

- Fix middots
- Some additional definition lists styling also have been removed as a result of the Vanilla update

## QA

- Open the https://ubuntu-com-9423.demos.haus and check copyright footer area and see the middots fixed.


## Issue
fix #9424
